src/Rfc6455/MessageHandler/WrongOpcodeHandler.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * This file is a part of Woketo package.
+ *
+ * (c) Nekland <dev@nekland.fr>
+ *
+ * For the full license, take a look to the LICENSE file
+ * on the root directory of this project
+ */
+
+namespace Nekland\Woketo\Rfc6455\MessageHandler;
+
+
+use Nekland\Woketo\Rfc6455\Frame;
+use Nekland\Woketo\Rfc6455\Message;
+use Nekland\Woketo\Rfc6455\MessageProcessor;
+use React\Socket\ConnectionInterface;
+
+class WrongOpcodeHandler implements Rfc6455MessageHandlerInterface
+{
+    public function supports(Message $message)
+    {
+        return in_array($message->getOpcode(), [3, 4, 5, 6, 7, 10, 11, 12, 13, 14, 15]);
```

Comment:
explain that it is the opcode reserved  for the future and they are considered wrong opcode for now only ? no ?